src/geometry/euclidean/basic.lean

```diff
@@ -729,24 +731,24 @@ def reflection (s : affine_subspace ℝ P) : P ≃ᵢ P :=
         (orthogonal_projection s p₁ -ᵥ p₁ +ᵥ orthogonal_projection s p₁ -ᵥ
          (orthogonal_projection s p₂ -ᵥ p₂ +ᵥ orthogonal_projection s p₂))
       = inner
-        ((_root_.orthogonal_projection s.direction (p₁ -ᵥ p₂)) +
-          _root_.orthogonal_projection s.direction (p₁ -ᵥ p₂) -
+        ((_root_.orthogonal_projection_real s.direction (p₁ -ᵥ p₂)) +
+          _root_.orthogonal_projection_real s.direction (p₁ -ᵥ p₂) -
```

Comment:
Is orthogonal projection a meaningful concept in complex inner product spaces or not? If it's meaningful, the renaming seems questionable. If it's not meaningful, the renaming should mean `_root_` is not needed anywhere to distinguish it from orthogonal projection in Euclidean affine spaces.

Reply:
It does make sense in the complex case. In fact I was planning to work on the orthogonal projection code next and properly generalize it. In the meantime I think you're right that we might as well just keep the old name.